docs/conf.py

```diff
@@ -12,23 +12,21 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import os
 import sys
-import sphinx_rtd_theme
-
-from forte import __version__
-
-# from unittest.mock import MagicMock
+import os
+#from unittest.mock import MagicMock
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 sys.path.insert(0, os.path.abspath('..'))
+from forte import __version__
+from forte.version import VERSION_SHORT as __version_short__
```

Comment:
Please group the imports together at the top of the file.

And please do not make changes to all the lines that are you haven't really touched below, such as those spaces after the #.

Reply:
Yes, I removed them in an early commit possibly in order to pass the pylint check. And I restore them in the latest commit.